docs/electron.rst

```diff
@@ -292,3 +304,14 @@ The preferred way of fixing this, is not a patch, but a build-time edit using ``
 .. code-block:: bash
 
   jq '.build.linux.target="dir"' <<<$(<package.json) > package.json
+
+Make setProgressBar and setBadgeCount work
+-------------------------------------------
+Electron has the 2 functions `setProgressBar <https://www.electronjs.org/de/docs/latest/api/browser-window#winsetprogressbarprogress-options>`_ and `setBadgeCount <https://www.electronjs.org/de/docs/latest/api/browser-window#winsetprogressbarprogress-options>`_.
+These functions allow showing a Progress bar and a Number in the Window Icon. It is implemented under Linux using the UnityLauncherAPI. This API is not implemented on every Desktop. A known Desktop who implements this is KDE.
+It is also implemented by the popular `Dash to Dock <https://micheleg.github.io/dash-to-dock>`_ GNOME extension and `Plank <https://launchpad.net/plank>`_.
+
+To make it work in Flatpak, the App needs to :ref:`use the correct desktop filename <use-correct-desktop-filename>`. The Flatpak also need the ``--talk-name=com.canonical.Unity``.
+
+Electron checks `checks if it's running on Unity or KDE <https://github.com/electron/electron/blob/fb88375ab4d2161dbf7e958a2a94c7c6d97dc84c/shell/browser/linux/unity_service.cc#L64>`_ before using the UnityLauncherAPI.
```

Comment:
Linkcheck doesn't like this link... not sure how best to resolve that.

Reply:
Is there maybe a way to whitelist this?

Reply:
I've not done it but maybe you can set `linkcheck_ignore = ['...']` in `conf.py`.

https://www.sphinx-doc.org/en/master/_modules/sphinx/builders/linkcheck.html

Reply:
Better to use anchors_ignore I think https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-linkcheck_anchors_ignore